### ISSUE TITLE
KeyError: `PROVIDE_AUTOMATIC_OPTIONS`

### ISSUE THREAD

**Issue by mark-szabo** [pallets/quart#389]
I'm getting this error since 0.19.9 came out:

```
2024-11-18T11:15:35.744598259Z File "/usr/src/app/app.py", line 886, in <module>
2024-11-18T11:15:35.744604859Z app = create_app()
2024-11-18T11:15:35.744763660Z ^^^^^^^^^^^^
2024-11-18T11:15:35.744775061Z File "/usr/src/app/app.py", line 46, in create_app
2024-11-18T11:15:35.744781761Z app = Quart(__name__)
2024-11-18T11:15:35.744787961Z ^^^^^^^^^^^^^^^
2024-11-18T11:15:35.744793561Z File "/usr/local/lib/python3.11/site-packages/quart/app.py", line 336, in __init__
2024-11-18T11:15:35.744799861Z self.add_url_rule(
2024-11-18T11:15:35.744806261Z File "/usr/local/lib/python3.11/site-packages/flask/sansio/scaffold.py", line 47, in wrapper_func
2024-11-18T11:15:35.744812261Z return f(self, *args, **kwargs)
2024-11-18T11:15:35.744818461Z ^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-18T11:15:35.744824561Z File "/usr/local/lib/python3.11/site-packages/flask/sansio/app.py", line 641, in add_url_rule
2024-11-18T11:15:35.744831161Z if "OPTIONS" not in methods and self.config["PROVIDE_AUTOMATIC_OPTIONS"]:
2024-11-18T11:15:35.744837261Z ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-18T11:15:35.744852561Z KeyError: 'PROVIDE_AUTOMATIC_OPTIONS'
2024-11-18T11:15:35.769508169Z [2024-11-18 11:15:35 +0000] [14] [INFO] Worker exiting (pid: 14)
```

Environment:

- Python version: 3.11-alpine
- Quart version: 0.19.9

**Comment by CannonWilson** [comment]
I'm running into this same issue

**Comment by davidism** [comment]
It sounds like you're not on 0.19.9. This was fixed in 0.19.9. Please verify that you've updated, and provide a minimal reproducible example if it's still an issue.
